src/views/NoticeOfWithdrawal.vue

```diff
@@ -423,7 +432,7 @@ export default class NoticeOfWithdrawal extends Mixins(CommonMixin, DateMixin, F
       // this is the id of filing being withdrawn, and of THIS filing
       // if filingID is 0, this is a new filing
       // otherwise it's a draft filing
-      this.filingToBeWithdrawn = this.$route.query.filingToBeWithdrawn as string || ''
+      this.filingToBeWithdrawn = Number(this.$route.query.filingToBeWithdrawn) || null
```

Comment:
Try this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus

Also, `NaN` is a special value to denote an invalid number, that you could also use if filingToBeWithdrawn isn't a route query parameter. (But no need to change here.)

Reply:
Looks OK!